### PR TITLE
Fixed Zenhub color issue

### PR DESCRIPTION
**Pull request by Geolim4**
Before:
screenshot_1

After:
screenshot_2

### CODE REVIEW

File: github-dark.css [hunk 1]
```diff
@@ -3585,6 +3585,17 @@
...
+  /* Issue/PR status text */
+  .zh-discussion-item .zh-pipeline-is,
+  .zh-discussion-item .zh-pipeline-was,
+  .zh-discussion-item .zh-estimate-is,
+  .zh-discussion-item .zh-estimate-was,
+  .zh-discussion-item .zh-issue,
+  .zh-discussion-item .zh-release{
+    color: #ccc !important;
+  }
+
```
Comment (Mottie) [inline github-dark.css]: Hi @Geolim4!

Thanks for contributing. Please combine these selectors with the block above with the same color definition. Wrap the selectors at around 80 columns as well. Thanks!

Comment (Mottie) [reply github-dark.css]: And remove the empty lines.

Comment (Geolim4) [reply github-dark.css]: Okay, I did not dare to do it :)

Comment (Mottie) [reply github-dark.css]: hehe, it's fine. Hopefully this was explained well enough in our [contributing guide](https://github.com/StylishThemes/GitHub-Dark/blob/master/.github/CONTRIBUTING.md)?

Comment (Geolim4) [reply github-dark.css]: @Mottie Done, commit added.

**Comment by Mottie** [comment]
Awesome, thanks! I haven't been using ZenHub much lately, so I haven't kept up with any changes.

**Comment by Geolim4** [comment]
I'm using it everyday on PhpFastCache so I can make you more PR like this in the future :)
Thanks for merging !

**Pull request merged**